src/azure.ts

```diff
@@ -2,7 +2,7 @@
 
```

Comment:
this file looks like an azure utility, shouldn't we move it to the folder `src/components/clusterprovider/azure`?